views/components/icon.erb

```diff
@@ -113,6 +113,11 @@
   <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="<%= classes %>">
     <path stroke-linecap="round" stroke-linejoin="round" d="M16.5 10.5V6.75a4.5 4.5 0 10-9 0v3.75m-.75 11.25h10.5a2.25 2.25 0 002.25-2.25v-6.75a2.25 2.25 0 00-2.25-2.25H6.75a2.25 2.25 0 00-2.25 2.25v6.75a2.25 2.25 0 002.25 2.25z" />
   </svg>
+<% when "hero-funnel" %>
+<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="w-6 h-6">
```

Comment:
![image](https://github.com/ubicloud/ubicloud/assets/2082070/fe5d8ea4-bfce-46f3-8cf1-f449d4db3aed)
How about this firewall icon?

```
  <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="<%= classes %>">
    <path stroke-linecap="round" stroke-linejoin="round" d="M 4 3 l 17 0 m -17 5 l 17 0 m -17 5 h 10.86 m 2.54 0 h 2.28 m -15.68 5 l 7.68 0 m -3.68 -15 l 0 5 m 0 5 l 0 5 m 4 -10 l 0 5 m 5 -10 l 0 5 m -13 -5 l 0 15 m 17 -15 l 0 8 m -4 9 m -1 3 c -1 0 -3 -1 -4 -3 c -1 -3 0.6667 -4.6667 1 -7 c 0 2 1 4 2 5 c -1 -3 0 -7 2 -10 c 0 3 0 6 2 8 c 0 -2 1 -4 2 -5 c 0 2 0.551 3.254 1 4 c 1.476 2.665 0.601 5.412 -0.973 7.004 c -1.54 1.207 -3 1 -5 1" />
  </svg>
```

Reply:
Woah I love this 🔥